glade_forms/lib/src/core/error/error_serverity.dart

```diff
@@ -0,0 +1,4 @@
+enum ErrorServerity {
```

Comment:
Nit: Having `Error` in the name is funny. Maybe `ValidationSeverity` or something like that?

Reply:
ve already use i.e. validationWarnings, validationErrors